frontend/TestRun/ResultsGraphs.svelte

```diff
@@ -172,48 +173,81 @@
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(1)}>Last Month</button>
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(3)}>Last 3 Months</button>
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(6)}>Last 6 Months</button>
+    <label class="mt-1">Time range:</label>
+    <div class="input-group input-group-inline input-group-sm mx-2">
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(1, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(1); showCustomInputs = false;}}>
+            Last Month
+        </button>
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(3, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(3); showCustomInputs = false;}}>
+            Last 3 Months
+        </button>
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(6, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(6); showCustomInputs = false;}}>
+            Last 6 Months
+        </button>
+        <button class="btn btn-outline-primary btn-sm {showCustomInputs ? 'active' : ''}"
+                on:click={() => { startDate = ''; endDate = ''; showCustomInputs = true; }}>Custom
+        </button>
+        {#if showCustomInputs}
+            <input type="date" class="form-control date-input" bind:value={startDate} on:change={() => fetchTestResults(test_id)}/>
+            <input type="date" class="form-control date-input" bind:value={endDate} on:change={() => fetchTestResults(test_id)}/>
+        {/if}
     </div>
-</div>
-<div class="filters-container">
-    {#each tableFilters as filterGroup}
-        {#each filterGroup.items as filter}
-            <button
-                    on:click={() => toggleTableFilter(filter, filterGroup.level)}
-                    class:selected={selectedTableFilters.some(f => f.name === filter)}
-                    style="background-color: {getTableFilterColor(filterGroup.level)}"
+    <label class="mt-1 ms-4">Releases:</label>
+    <div class="input-group input-group-inline input-group-sm mx-2">
+        {#each Object.keys(releasesFilters) as release}
+            <button class="btn btn-sm btn-outline-dark"
+                    on:click={() => toggleReleaseFilter(release)}
+                    class:active={releasesFilters[release]}
             >
-                {filter}
+                {release}
             </button>
         {/each}
-    {/each}
-    {#each columnFilters as filter}
-        <button
-                on:click={() => toggleColumnFilter(filter)}
-                class:selected={selectedColumnFilters.some(f => f === filter)}
-                style="background-color: #a3e2cc"
-        >
-            {filter}
+    </div>
+</div>
+
+<label>Tables Filters</label>
```

Comment:
`<label>` should not be used like that, use either heading `hN` (e.g. `h4`) or a normal `div`

Reply:
I replaced it with <span>